packages/@react-spectrum/tabs/stories/Tabs.stories.js

```diff
@@ -22,74 +22,74 @@ import {Tabs} from '..';
 storiesOf('Tabs', module)
   .add(
     'Default',
-    () => render()
+    () => render({'aria-label': 'Default'})
```

Comment:
We should probably add a console warning if no label is provided like we do for other components to make this easier to catch.

Reply:
A tablist doesn't require an `aria-label` attribute, but without an ancestor landmark element like `<main>`, which is missing in our Storybook example pages, I noticed that with VoiceOver the first tab to receive focus announces not only the focused tab, but all the other tabs as well. VoiceOver is announcing the nearest named container element or landmark region to provide context. If we add a `<main>`, VoiceOver announces the focused tab and the `<main>` landmark for context. However, with no `aria-label` or ancestor landmark, VoiceOver announces the contents of the tablist for context.

Reply:
Should we place a `main` tag in our stories then? or how do we expect people to use tabs? And what do we want to do in our documentation site to communicate this? I don't think we can have a bunch of `main`'s?

Reply:
We can the main tag to the storybook story template.